### PR TITLE
Remove not null

### LINKED ISSUE #42: Add Description to Games Table
## Description
Add a Description column to the ERD and to schema.sql

## Required Files
- schema.sql
- ERD Diagram.png

## User Story
As a subscriber I need to know a bit about the games so I can stay informed on latest releases.

### PR DESCRIPTION
**Pull request by annalisev**
Allows the publisher column to be null.

Closes #42

### CODE REVIEW

File: pipeline/extract/extract_gog.py [hunk 1]
```diff
@@ -12,28 +14,92 @@ def scrape_from_game_page(web_address: str) -> list:
...
+    # # print(dev, pub, get_platform_ids(platform), date)
+
+
+def get_soup(web_address: str):
+    '''Returns a soup object for a game given the web address.'''
+    response = req.get(web_address)
+    return BeautifulSoup(response.text, features="html.parser")
+
+
+def get_detail_links(game_soup) -> list:
```
Comment (hhm970) [inline pipeline/extract/extract_gog.py:41]: I think `game_soup` should be a `BeautifulSoup` object

File: pipeline/extract/extract_gog.py [hunk 2]
```diff
@@ -12,28 +14,92 @@ def scrape_from_game_page(web_address: str) -> list:
...
     response = req.get(web_address)
     game_soup = BeautifulSoup(response.text, features="html.parser")
+
+    scripts = (game_soup.find_all(
+        'script'))
+
+    filtered_scripts = [
+        t for t in scripts if 'window.productcardData.cardProductSystemRequirements' in str(t)][0]
+    j = list(json.loads((filtered_scripts).text.split(
+        'window.productcardData.cardProductSystemRequirements')[-1][3:].split('window.productcardData.cardProductPromoEndDate')[0].split(';')[0]).keys())
```
Comment (shindym) [inline pipeline/extract/extract_gog.py:24]: Why is this line so long?

File: pipeline/extract/extract_gog.py [hunk 3]
```diff
@@ -12,28 +14,92 @@ def scrape_from_game_page(web_address: str) -> list:
...
+        'a', class_='details__link')
+
+
+def get_json(game_soup) -> dict:
+    '''Returns a JSON object about a given game soup.'''
+    return json.loads(game_soup.find(
+        'script', type='application/ld+json').text)
+
+
+def get_developer(links: list) -> str:
```
Comment (shindym) [inline pipeline/extract/extract_gog.py:54]: Good use of typehints

**Comment by annalisev** [comment]
I've updated this!

**Review by shindym — APPROVED** [review]
Looks all in order!

**Review by hhm970 — APPROVED** [review]
Looks all good to me!

**Review by Eibironke — APPROVED** [review]
🥇

**Pull request merged**